src/OrchardCore.Modules/OrchardCore.Setup/GraphQL/GraphQLSchemaHashService.cs

```diff
@@ -0,0 +1,13 @@
+using System.Threading.Tasks;
+using OrchardCore.Apis.GraphQL;
+
+namespace OrchardCore.Contents.GraphQL
+{
+    public class GraphQLSchemaHashService : IGraphQLSchemaHashService
```

Comment:
Please see my comments in OrchardCore.Apis.GraphQL/Services/SchemaService.cs regarding to IGraphQLSchemaHashService

Reply:
argh!! Cant find your comments?!?!?!? any ideas how to get to them?